intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -81,9 +81,11 @@ static bool kv_cache_init(const struct model_hparams& hparams, struct model_kv_c
   }
 
   // NE_TYPE_JBLAS can not be allocated memory
-  cache.k = ne_new_tensor_1d(cache.ctx, wtype == NE_TYPE_JBLAS ? NE_TYPE_I8 : wtype, n_elements_k, NE_SIZE_CALC);
+  cache.k = ne_new_tensor_1d(cache.ctx, wtype == NE_TYPE_JBLAS ? NE_TYPE_I8 : wtype, n_elements_k + 64, NE_SIZE_CALC);
```

Comment:
make 64 as an static variable or get from some static function

Reply:
Changed in 79acbd5. (Using name inspired from `intel_extension_for_transformers/llm/runtime/deprecated/executor/include/i_malloc.hpp`)